ts/src/test/test.ts

```diff
@@ -230,44 +229,31 @@ export default class testMainClass extends baseMainTestClass {
             return;
         }
         let skipMessage = undefined;
-        const isFetchOhlcvEmulated = (methodName === 'fetchOHLCV' && exchange.has['fetchOHLCV'] === 'emulated'); // todo: remove emulation from base
-        if (!isLoadMarkets && (!(methodName in exchange.has) || !exchange.has[methodName]) || isFetchOhlcvEmulated) {
+        if (!isLoadMarkets && (!(methodName in exchange.has) || !exchange.has[methodName])) {
             skipMessage = '[INFO:UNSUPPORTED_TEST]'; // keep it aligned with the longest message
         } else if ((methodName in this.skippedMethods) && (typeof this.skippedMethods[methodName] === 'string')) {
             skipMessage = '[INFO:SKIPPED_TEST]';
         } else if (!(methodNameInTest in this.testFiles)) {
             skipMessage = '[INFO:UNIMPLEMENTED_TEST]';
         }
-        const argsStringified = '(' + args.join (',') + ')';
-        try {
```

Comment:
try/catch here was solely for catching `AuthenticationError`, so, I've moved that into `testSafe`, where all exceptions are being handled (centralized)